hist/hist/src/TH3.cxx

```diff
@@ -2362,7 +2362,15 @@ TH2D *TH3::DoProject2D(const char* name, const char * title, const TAxis* projX,
 
 TH1 *TH3::Project3D(Option_t *option) const
 {
-   TString opt = option; opt.ToLower();
+   TString opt = option;
+   TString extra_name = option;
+   Int_t underscore = extra_name.Index("_");
```

Comment:
I think it is better to use the "Last" function to find the last accurence of "_". The user provided name could have an "_" 
while not the option string.